ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml

```diff
@@ -199,7 +199,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-capability-baselinecaps-vset-additionalcaps
-- as: azure-ipi-baselinecaps-vset-amd-f28
+- as: azure-ipi-baselinecaps-vset-amd-f28-destructive
   cron: 23 16 30 * *
```

Comment:
cron entry is not updated

Reply:
for 4.12, the both frequency are f28, so don't need to update cron, rignt?

Reply:
cron entry is based on job name, since job name is changed here, so it's better to update cron entry accordingly